Packs/CadoResponse/Integrations/CadoResponse/CadoResponse.py

```diff
@@ -0,0 +1,493 @@
+                                   file_name: Optional[str]) -> Dict[str, Any]:
+        ''' Calls the POST /api/v2/projects/{id}/imports/s3 endpoint to
+            trigger an acquisition of a given bucket or file
+
+            :param Optional[int] project_id: The ID of the project you wish to attach the acquisition to
+            :param Optional[str] bucket: The S3 bucket name containing the file
+            :param str file_name: The name of the file to process
+
+            :return JSON response from /projects/{id}/imports/ec2 endpoint
+            :rtype Dict[str, Any]
+        '''
+
+        if not project_id:
+            project_id = demisto.params().get('CadoResponse_DefaultProject', 1)
+
+        if not bucket:
+            bucket = demisto.params().get('CadoResponse_DefaultBucket', 'cado-default-bucket')
+
+        payload: Dict[str, Any] = {
+            'bucket': bucket,
+            'file_name': file_name
+        }
+
+        return self._http_request(
+            method='POST',
+            url_suffix=f'/projects/{project_id}/imports/s3',
+            json_data=payload
+        )
+
+
+''' Command Line Handlers '''
+
+
+def test_module(client: Client) -> str:
+    ''' Command handler for !test-module '''
+
+    result: Dict[str, Any] = client.heartbeat()
+    status: Optional[str] = result['status']
+
+    if status is not None and status == 'Running':
+        return 'ok'
+
+    return 'Cado Response is not running'
+
+
+def create_project_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ''' Command handler for cado-create-project '''
+
+    unix_timestamp: str = str(int(time.time()))
+    project_name: str = args.get('project_name', unix_timestamp)
+    project_description: Optional[str] = args.get('project_description', None)
+    result: Dict[str, Any] = client.create_project(project_name, project_description)
+
+    return CommandResults(
+        outputs_prefix='CadoResponse.Project',
+        outputs_key_field='id',
+        outputs=result
+    )
+
+
+def list_project_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ''' Command handler for cado-list-project '''
+
+    project_id: Optional[int] = args.get('project_id', None)
+    limit: int = int(args.get('limit', 50))
+
+    if project_id:
+        result: Any = client.get_project(project_id)
+    else:
+        result = client.list_projects(limit)
+
+    return CommandResults(
+        outputs_prefix='CadoResponse.Projects',
+        outputs_key_field='id',
+        outputs=result
+    )
+
+
+def get_pipeline_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ''' Command handler for cado-get-pipeline '''
+
+    project_id: Optional[int] = args.get('project_id', None)
+    limit: int = int(args.get('limit', 50))
+    pipeline_id: Optional[int] = args.get('pipeline_id', None)
+
+    if pipeline_id:
+        result: CadoResponseCombinedOutput = client.get_pipeline(pipeline_id, project_id)
```

Comment:
same thing as the comment above, we'd want to return a readable output instead of an error, in case the pipeline with the given id does not exist